tests/test_cases/test_iteration_verilog/Makefile

```diff
@@ -52,3 +52,7 @@ include $(COCOTB)/makefiles/Makefile.inc
 include $(COCOTB)/makefiles/Makefile.sim
 
 endif
+
+ifeq ($(SIM),ius)
+		EXTRA_ARGS += -licqueue
+endif
```

Comment:
How about adding the `-licqueue` argument to the simulator Makefile to always use it? I can't think of many use cases where you don't want that flag, it seems to be a sensible default. And having such simulator-specific things within the tests doesn't sound right to me.

Reply:
We're unable to test whether this works for other simulators - so we can't guarantee that the flag works in every simulator. Unless you mean you want it to be moved to the `Makefile.ius`?